client/src/components/app.jsx

```diff
@@ -34,8 +37,43 @@ class App extends React.Component {
         this.setState({
           item: itemData
         });
+      })
+      .catch((error) => {
+        console.log(error);
       });
+    this.getPrice(randomId);
+  }
+
+  getPrice(id) {
+    let productId = id;
+    axios.get(`http://localhost:4003/priceandinventory/id/${productId}`)
+      .then((response) => {
+        console.log(response);
+        let itemPrice = response.data[0].price;
+        this.setState({
+          price: itemPrice
+        });
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  }
 
+  //handle color box click
+  handleColorBoxClick (id) {
+    //will make call to /description/${productId}
+    let productId = id;
+    axios.get(`http://localhost:4004/description/${productId}`)
+      .then((response) => {
+        let itemData = response.data[0];
+        this.setState({
+          item: itemData
+        })
+          .catch((error) => {
+            console.log(error);
+          });
+      });
+    this.getPrice(productId);
   }
 
   render () {
```

Comment:
On line 79 and a half you could for example declare a `let` variable with the name `item` with the assigned value of `this.state.item`. This way you can pass as prop for example `item.brand` or `item.color`. This is not a big deal at all, it just saves you from writing `this.state.item` for every prop you pass to a component.

Reply:
I will go ahead and change this on my next PR!